app/javascript/react/components/NewReviewForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+
+import ErrorList from './ErrorList'
+
+const NewReviewForm = props => {
+  const game_id = props.game_id
+  const [review, setReview] = useState({
+    rating: "",
```

Comment:
I would set this to null instead of a string by default

Reply:
We originally were going to set it to null, but got errors in the console about setting a form value to null when the page first renders